website/docs/r/storage_account_local_user.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_account_local_user"
+description: |-
+  Manages a Storage Account Local User.
+---
+
+# azurerm_storage_account_local_user
+
+Manages a Storage Account Local User.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "WestEurope"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "example-account"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  is_hns_enabled           = true
+}
+
+resource "azurerm_storage_container" "example" {
+  name                 = "example-container"
+  storage_account_name = azurerm_storage_account.example.name
+}
+
+resource "azurerm_storage_account_local_user" "example" {
+  name                 = "user1"
+  storage_account_id   = azurerm_storage_account.example.id
+  ssh_key_enabled      = true
+  ssh_password_enabled = true
+  home_directory       = "example_path"
+  ssh_authorized_key {
+    description = "key1"
+    key         = local.first_public_key
+  }
+  ssh_authorized_key {
+    description = "key2"
+    key         = local.second_public_key
+  }
+  permission_scope {
+    permissions {
+      read   = true
+      create = true
+    }
+    service       = "blob"
+    resource_name = azurerm_storage_container.example.name
```

Comment:
```suggestion
    resource_name = "example-container"
```

resource [`azurerm_storage_container`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container) doesn't have an attribute "name", so this needs to be updated.

Reply:
This one? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container#name

Reply:
Yes, that is an argument, not an attribute (input not output)